contracts/upgradeable_contracts/erc20_to_native/FeeManagerErcToNative.sol

```diff
@@ -0,0 +1,20 @@
+pragma solidity 0.4.24;
+
+import "../BaseFeeManager.sol";
+import "../../IBlockReward.sol";
+
+contract FeeManagerErcToNative is BaseFeeManager {
+
+    function blockRewardContract() internal view returns(IBlockReward) {
+        return IBlockReward(addressStorage[keccak256(abi.encodePacked("blockRewardContract"))]);
+    }
+
+    function onAffirmationFeeDistribution(address _rewardAddress, uint256 _fee) internal {
+        IBlockReward blockReward = blockRewardContract();
+        blockReward.addExtraReceiver(_fee, _rewardAddress);
+    }
+
+    function onSignatureFeeDistribution(address _rewardAddress, uint256 _fee) internal {
+        _rewardAddress.send(_fee);
```

Comment:
Should we use `send` or `transfer` in this case?

Reply:
Thanks! Applied same approach here https://github.com/poanetwork/poa-bridge-contracts/pull/132/commits/51708637ac1a46d1657f5fd588a1814685c60e89